src/main/java/com/rtb/projectmanagementtool/user/CreateUserServlet.java

```diff
@@ -51,11 +69,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // get/create cookie and set value to userID
     auth.setLoggedInCookie(request, response, userID);
 
-    // redirect to user_settings
-    response.sendRedirect("/user_settings.html");
+    // redirect them to their new profile
+    response.sendRedirect("/user_profile.jsp");
+  }
 
-    // get the form data from the parameters and set
-    // them into the blank UserData before pushing into datastore
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+    // Get login URL
+    request.setAttribute(
```

Comment:
This code is correct, just in the wrong spot.  The login page you see with the two buttons is created by LoginServlet, so you'd want to add info for this button in that servlet:
https://github.com/googleinterns/step171-2020/blob/master/src/main/java/com/rtb/projectmanagementtool/auth/LoginServlet.java


Then I guess the next question would be what you'll need to do in doGet for CreateUserServlet.  (Optional) You can validate if they're logged in via UserService and do an auto redirect if they're not, in case they manually type in the "site.com/create-user-url" instead of going through the button.  Besides that I think you'll just need to redirect to create-user.jsp to show the form